examples/resources/awscc_route53_hosted_zone/awscc_route53_hosted_zone_private.tf

```diff
@@ -0,0 +1,23 @@
+resource "awscc_route53_hosted_zone" "this" {
+  name = "this.com"
+  hosted_zone_tags = [
+    {
+      key   = "Name"
+      value = "this"
+    },
+    {
+      key   = "Modified By"
+      value = "AWSCC"
+    }
+  ]
+  vp_cs = [
```

Comment:
this part doesnt read right, lets investigate this a bit

Reply:
https://github.com/hashicorp/terraform-provider-awscc/blob/2e5e4089ce02498d5dbfb40d5e2900831b284f8a/internal/aws/route53/hosted_zone_resource_gen.go#L237C4-L237C9

Reply:
@ewbankkit , is there a overrides that explain why this attribute map from "VPC" into "vp_cs"